sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/cli/OwaspZapScanExecutor.java

```diff
@@ -39,15 +38,29 @@ public void execute(OwaspZapScanContext scanContext) throws ZapWrapperRuntimeExc
 
     }
 
+    private void assertTargetReachable(OwaspZapScanContext scanContext) {
+        boolean isReachable = false;
+        for (String url : scanContext.getOwaspZapURLsIncludeList()) {
+            URI uriToCheck = URI.create(url);
+            if (connectionChecker.isTargetReachable(uriToCheck, scanContext.getProxyInformation())) {
```

Comment:
There should be a retry logic, who knows maybe the URL is only temporarily not reachable (high load, too many requests etc.).